src/sentry/workflow_engine/models/detector.py

```diff
@@ -131,5 +131,4 @@ def enforce_config_schema(sender, instance: Detector, **kwargs):
         raise ValidationError("Detector config must be a dictionary")
 
     config_schema = group_type.detector_config_schema
-    if instance.config:
```

Comment:
We should be validating the config always, since some fields are required and an empty config means we don't have those fields